crates/ruff_linter/src/rules/refurb/rules/slice_to_remove_prefix_or_suffix.rs

```diff
@@ -0,0 +1,498 @@
+use crate::{checkers::ast::Checker, settings::types::PythonVersion};
+use ruff_diagnostics::{AlwaysFixableViolation, Diagnostic, Edit, Fix};
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::{self as ast};
```

Comment:
teeny tiny nit:

```suggestion
use ruff_python_ast as ast;
```

Reply:
Well if you leave it around that's how you get lice! Nit picked